kotest-assertions/kotest-assertions-core/src/jvmMain/kotlin/io/kotest/matchers/bigdecimal/matchers.kt

```diff
@@ -43,6 +43,16 @@ fun beInClosedRange(range: ClosedRange<BigDecimal>) = object : Matcher<BigDecima
    )
 }
 
+infix fun BigDecimal.shouldBeInOpenEndRange(range: OpenEndRange<BigDecimal>) = this should beInOpenEndRange(range)
+infix fun BigDecimal.shouldNotBeInOpenEndRange(range: OpenEndRange<BigDecimal>) = this shouldNot beInOpenEndRange(range)
+fun beInOpenEndRange(range: OpenEndRange<BigDecimal>) = object : Matcher<BigDecimal> {
+   override fun test(value: BigDecimal) = MatcherResult(
+      range.contains(value),
+      { "Value $value should be in range from ${range.start} to ${range.endExclusive} (Exclusive)" },
+      { "Value $value should not be in range from ${range.start} to ${range.endExclusive} (Exclusive)" }
```

Comment:
Do you think it would be important to also add '(Inclusive)' to the start of the range?

Reply:
> Do you think it would be important to also add '(Inclusive)' to the start of the range?

sure, thx for the idea